src/components/Albums/AlbumForm.vue

```diff
@@ -294,6 +294,7 @@ export default {
 	.form-buttons {
 		display: flex;
 		justify-content: space-between;
+		flex-direction: column;
```

Comment:
always? I thought only on mobile?

Reply:
![Screenshot from 2023-10-13 11-18-04](https://github.com/nextcloud/photos/assets/6078378/6930be46-46be-4b95-aaf4-decbdfc477a8)

desktop

Reply:
Or mybe just use flex-wrap? Wdyt?

Reply:
```suggestion
		flex-wrap: wrap;
```

Reply:
@szaimen nope ;(
![Screenshot from 2023-10-16 15-56-37](https://github.com/nextcloud/photos/assets/6078378/b696cc8c-059f-4fdc-bec8-6ed35db446d7)

Reply:
Hm... Is there some reason why that does not work? Should usually be shown in the inspector tools...

Reply:
does it works for you? For me: styles are there in dev tools.